packages/asc-ui/src/components/StepByStepNav/StepByStepNav.tsx

```diff
@@ -0,0 +1,50 @@
+import type { FunctionComponent, HTMLAttributes } from 'react'
+import type { StepByStepNavProps } from './StepByStepNavStyle'
+
+import StepByStepNavStyle from './StepByStepNavStyle'
+import Typography from '../Typography'
+
+const StepByStepNav: FunctionComponent<
+  StepByStepNavProps & HTMLAttributes<HTMLElement>
+> = (props) => {
+  const { activeItem, steps, stepsCompleted } = props
+  let realActiveItem: number
+
+  if (!activeItem || activeItem < 1) {
+    realActiveItem = 1
+  } else if (stepsCompleted || activeItem > steps.length) {
+    realActiveItem = steps.length
+  } else {
+    realActiveItem = activeItem
+  }
+
+  return (
+    <StepByStepNavStyle
+      {...props}
+      activeItem={realActiveItem}
+      role="group"
+      aria-label="progress"
+    >
+      <ol>
+        {steps.map(({ label }, index) => (
+          <li
+            key={label}
+            aria-current={
+              !stepsCompleted && index === realActiveItem - 1 && 'step'
+            }
+          >
+            <Typography as="span">{label}</Typography>
+          </li>
+        ))}
+      </ol>
+    </StepByStepNavStyle>
+  )
+}
+
+StepByStepNav.defaultProps = {
```

Comment:
I'm seeing some ESLint errors in webstorm. Is this on my end only?
![image](https://user-images.githubusercontent.com/1095870/143447230-4a8b6502-e603-4520-949e-ac83404a5916.png)